src/schemas/doodadsSchema.ts

```diff
@@ -0,0 +1,52 @@
+import { gql } from "graphql-tag";
+
+export const doodadsSchema = gql`
+  type Doodad {
+    id: ID!
+    name: String!
+    description: String!
+    price: Float!
+    image: String!
+    created: String!
+    modfied: String!
+  }
+
+  extend type Query {
+    doodads: Doodads!
+  }
+
+  input DoodadFilterInput {
+    keywords: String
+    minPrice: Float
+    max_Price: Float
```

Comment:
Keep it consistent with variable names
```suggestion
    maxPrice: Float
```